r/tests/testthat/test-Array.R

```diff
@@ -727,6 +727,15 @@ test_that("[ accepts Arrays and otherwise handles bad input", {
   )
 })
 
+test_that("[ %in% looks up string key in dictionary", {
+  a1 <- Array$create(as.factor(c("A", "B", "C")))
+  a2 <- DictionaryArray$create(c(0L, 1L, 2L), c(4.5, 3.2, 1.1))
+  arrow:::call_function("is_in_meta_binary", a1, Array$create("B"))
+  arrow:::call_function("is_in_meta_binary", a1, Array$create("D"))
+  arrow:::call_function("is_in_meta_binary", a2, Array$create(3.2))
```

Comment:
Shouldn't you check return values here?

Reply:
I was looking to test for errors but the added check is better. Adding.